<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Biomarkers in Clinical Trials</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/ionicons-2.0.1/css/ionicons.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>


<style type="text/css">
  p.abstract{
    text-align: center;
    font-weight: bold;
  }
  div.abstract{
    margin: auto;
    width: 90%;
  }
</style>



<link rel="stylesheet" href="custom-css.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Biomarkers of Clinical Trials</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="Biomarkers.html">Biomarkers</a>
</li>
<li>
  <a href="Conditions.html">Conditions</a>
</li>
<li>
  <a href="Pairs.html">Top Pairs</a>
</li>
<li>
  <a href="Liver.html">Liver Failure</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com">
    <span class="ion ion-social-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Biomarkers in Clinical Trials</h1>
<h4 class="author">Janet Piñero</h4>
<address class="author_afil">
MedBioinformatics Solutions SL, Almogàvers 165, 08018 Barcelona, Spain; Research Programme on Biomedical Informatics (GRIB), IMIM (Hospital del Mar Medical Research Institute), UPF (Pompeu Fabra University), Dr. Aiguader, Barcelona, Spain<br><a class="author_email" href="mailto:#"><a href="mailto:janet.pinero@disgenetplus.com">janet.pinero@disgenetplus.com</a></a>
</address>
<h4 class="author">Pablo S. Rodriguez</h4>
<address class="author_afil">
Research Programme on Biomedical Informatics (GRIB), IMIM (Hospital del Mar Medical Research Institute), UPF (Pompeu Fabra University), Dr. Aiguader, Barcelona, Spain<br><a class="author_email" href="mailto:#"><a href="mailto:prodriguez1@imim.es">prodriguez1@imim.es</a></a>
</address>
<h4 class="author">Francesco Ronzano</h4>
<a class="author_email" href="mailto:#"><a href="mailto:francesco.ronzano@upf.edu">francesco.ronzano@upf.edu</a></a>
</address>
<h4 class="author">Jordi Valls-Margarit</h4>
<address class="author_afil">
MedBioinformatics Solutions SL, Almogàvers 165, 08018 Barcelona, Spain<br><a class="author_email" href="mailto:#"><a href="mailto:jordi.valls@disgenetplus.com">jordi.valls@disgenetplus.com</a></a>
</address>
<h4 class="author">Pablo Acuosto</h4>
<address class="author_afil">
Research Programme on Biomedical Informatics (GRIB), IMIM (Hospital del Mar Medical Research Institute), UPF (Pompeu Fabra University), Dr. Aiguader, Barcelona, Spain<br><a class="author_email" href="mailto:#"><a href="mailto:pablo.acuosto@upf.edu">pablo.acuosto@upf.edu</a></a>
</address>
<h4 class="author">Ferran Sanz</h4>
<address class="author_afil">
MedBioinformatics Solutions SL, Almogàvers 165, 08018 Barcelona, Spain; Research Programme on Biomedical Informatics (GRIB), IMIM (Hospital del Mar Medical Research Institute), UPF (Pompeu Fabra University), Dr. Aiguader, Barcelona, Spain<br><a class="author_email" href="mailto:#"><a href="mailto:ferran.sanz@disgenetplus.com">ferran.sanz@disgenetplus.com</a></a>
</address>
<h4 class="author">Laura Furlong</h4>
<address class="author_afil">
MedBioinformatics Solutions SL, Almogàvers 165, 08018 Barcelona, Spain; Research Programme on Biomedical Informatics (GRIB), IMIM (Hospital del Mar Medical Research Institute), UPF (Pompeu Fabra University), Dr. Aiguader, Barcelona, Spain<br><a class="author_email" href="mailto:#"><a href="mailto:laura.furlong@disgenetplus.com">laura.furlong@disgenetplus.com</a></a>
</address>
<div class="abstract">
<p class="abstract">Abstract</p>
<p>Over 95% of protein-coding genes are associated with disease, however, only a dozen proteins are recognized as FDA qualified biomarkers. For this reason there is an opportunity to use this knowledge to find new biomarkers that can be translated to clinics in order to improve disease diagnosis and treatment, and monitor disease progression. Although a few resources propose candidate biomarkers of disease based on the scientific literature, none of them identify biomarkers being tested in clinical studies, such as the ones collected in clinicaltrials.gov. Clinical studies provide evidence to assess the value of a biomarker in the clinical setting. Nevertheless, the data about biomarkers are coded as free text and not standardized, posing challenges in finding, analyzing, and processing these data by computational tools. We have implemented a text mining pipeline to identify protein and gene biomarkers assessed in clinical trials in the context of disease and therapeutic intervention. The pipeline includes a classification system to describe the different ways in which biomarkers are measured. We found over 3,000 biomarkers measured in over 43,000 Clinical Trials involving 2,600 conditions. Our data analytics pipeline allows assessing biomarker specificity and discovering emerging trends about biomarkers of disease. The resulting dataset enables a new way of explorating the data in clinicaltrials.gov, integrating the drug, and disease information with the normalized data for biomarkers. For instance, it can be used to answer different questions, such as what are the biomarkers used to monitor specific conditions, what are the most used biomarkers, what types of biomarkers (phosphorylation, genetic) are used for a particular condition, among other applications.</p>
</div>

</div>


<div id="introduction" class="section level1">
<h1>Introduction</h1>
<p>In the last two decades, biomarkers have gained prominence in drug discovery, development, and approval processes <span class="citation">(Gromova et al. 2020)</span>. Biomarkers may also provide insights on disease prognosis, progression, and may be used to monitor the efficacy and safety of the therapy. Biomarkers may be simple molecules such as metabolites (blood lactate levels following exercise), carbohydrates (glucose for diabetes mellitus or galactomannan, used to diagnose invasive aspergillosis infections in humans), or peptides, and proteins (for example insulin, hemoglobin, prostate-specific antigen, and C-reactive protein). Biomarkers also include cells, histological data, and genetic markers. Genetic markers encompass single genes, small collections of three or four genes, and large collections (arrays) of genes (For example the pam50 signature for breast cancer <span class="citation">(Bernard et al. 2009)</span>. Genetic markers include classical mRNA, as well as micro-RNA (miRNA). More complex biomarkers are cells like platelets and T cells, and autoantibodies, or even physiological variables, like blood pressure.</p>
<p>Typically, biomarkers can be measured in the blood or other biological fluids. Nevertheless, more recently, the detection of alterations in the mRNA levels of genes, or their methylation level, and the presence of SNPs have emerged as new types of biomarkers <span class="citation">(Sinnott-Armstrong et al. 2021; Sorokin et al. 2020; Miura, Hasegawa, and Shiota 2008; Bock et al. 2016)</span>.<br />
Biomarkers are key measurements of outcomes both in basic research and clinical studies, and are frequently used in clinical studies <span class="citation">(Strimbu and Tavel 2010)</span> as “surrogate endpoints”. A surrogate endpoint is a biomarker that is intended to serve as a substitute for a clinically meaningful endpoint and is expected to predict the effect of a therapeutic intervention. While numerous laboratory biomarkers may be associated with a particular disease state, the term ‘surrogate’ indicates the ability of a biomarker to provide information about the clinical prognosis or efficacy of a therapy.<br />
The <a href="https://clinicaltrials.gov/">Clinical Trials database</a> (CT.gov from now on) is a rich source of information about the use of biomarkers in clinical studies. Nevertheless, the biomarkers measured in clinical trials are mentioned as free text and not standardized, posing challenges in finding, analyzing, and processing these data by computational tools. Text mining approaches may assist in the identification of biomarkers used in a therapeutic area or a specific disease, discover biomarkers trends across all the clinical studies recorded at CT.gov, or assess the specificity of a biomarker, among other applications. The case for using text mining to standardize and normalize the information in clinical trials is particularly compelling as the industry looks to cut the costs and time required for clinical studies.</p>
<p>In this contribution, we present the results of text mining CT.gov records to identify and standardize information about disease biomarkers. For this, we have developed a pipeline to identify biomarkers mentioned in different sections of a clinical trial, classify them into different categories. We have created a corpus of CT.gov records annotated with different biomarker types, which was used to train a machine learning model that classifies biomarkers according to the following categories: (1) protein biomarkers, which encompasses proteins that are measured in blood, or other biological fluids, or that have been detected using immunohistochemistry, (2) genetic biomarkers, which comprise genes that contain polymorphisms, copy number alterations, or chromosomal abnormalities, (3) phosphobiomarkers, that involve abnormal protein phosphorylation, (4) epigenetic biomarkers, that undergo changes in DNA methylation patterns, and (5) expression biomarkers, which are genes that are measured as mRNA, and non-coding RNA such as microRNAs, and (6) cell markers, mainly composed of cell surface antigens. The resulting dataset enables a new way of exploring clinicaltrials.gov that in novel ways, for instance, it can be used to answer different questions, for example which biomarkers are used to monitor a specific condition, what are the most used biomarkers, or types of biomarkers (phosphorylation, genetic), among other applications.</p>
</div>
<div id="methods" class="section level1">
<h1>Methods</h1>
<p>We downloaded the Clinical trials data from the <a href="https://aact.ctti-clinicaltrials.org/download">Clinical Trials Transformation Initiative</a>, on December 17, 2021. We performed named entity recognition to detect mentions of genes in the outcomes, outcome measurements, and design outcomes sections of the Clinical Trials.</p>
<p>Clinical trials provide the name of the conditions investigated in the trial. Conditions are the diseases, disorders, syndromes, illnesss, or injuries being studied, that are annotated using an NLM algorithm <span class="citation">(Tasneem et al. 2012)</span>. We used the conditions annotated by the algorithm, and we normalized the names provided in the files to MeSH identifiers using the UMLS Metathesaurus <span class="citation">(Bodenreider 2004)</span>.</p>
<p>We used the <a href="https://beta.disgenetplus.com/">DISGENET plus</a> text mining tool <span class="citation">(Piñero et al. 2019)</span> that improves the detection and normalization of mentions of genes by relying on deep learning models complemented by an extensive set of custom heuristics, and by integrating analysis modules to consistently detect and characterize distinctive linguistic traits of biomedical texts like the use of acronyms.</p>
<p>We then selected 1,000 sentences with mentions of biomarkers, and classified them into 6 different categories: (1) protein biomarkers, which encompasses proteins that are measured in blood, or other biological fluids, or that have been detected using immunohistochemistry, (2) genetic biomarkers, which includes genes that contain polymorphisms, copy number alterations, or chromosomal abnormalities, (3) phosphobiomarkers, that include abnormal protein phosphorylation, (4) epigenetic biomarkers, that undergo changes in DNA methylation patterns, and (5) expression biomarkers, which are genes that are measured as mRNA, and non-coding RNA such as microRNAs, and (6) cell markers, mainly composed of cell surface antigens. We used this corpus to train a random forest classifier to then assign the biomarker types to the rest of the data. We use the rpart package, via the library <a href="https://www.tidymodels.org/">tidymodels</a> in R. We performed hyperparameter tuning for the number of tokens (max_tokens), trees (trees), members (min_n), and features (mtry) using a regular grid. We provide more details of the implementation, along with a sample of the developed corpus in <a href="https://github.com/jpinero/biomarkers">this github repositoy</a>.</p>
<p>To explore this dataset we built a shiny app (version?), a web interface that will enable searching, filtering and browsing the data, and with visualization capabilities. We will make available the App at <a href="https://beta.disgenetplus.com/biomarkers" class="uri">https://beta.disgenetplus.com/biomarkers</a>.</p>
</div>
<div id="results" class="section level1">
<h1>Results</h1>
<p>In Figure 1 we show a general scheme of the pipeline.</p>
<div class="figure">
<img src="images/Figures_summary.png" alt="Figure 1: The Pipeline." width="100%" />
<p class="caption">
Figure 1: The Pipeline.
</p>
</div>
<p>Using this pipeline, we have processed 398,516 clinical studies available at CT.gov, carried out between 1999 and 2021. We found over 43,000 clinical studies measuring 3,100 biomarkers (see the <a href="Biomarkers.html">top 100 biomarkers</a>) for 2,600 diseases (see the <a href="Conditions.html">top 100 conditions</a>). Figure 2 shows the trends in the use of biomarker across all therapeutic areas in these years.</p>
<div class="figure">
<img src="images/Figure_therapeutic_area_conditions.png" alt="Figure 2: Clinical Trials containing biomarkers by therapeutic area per year." width="100%" />
<p class="caption">
Figure 2: Clinical Trials containing biomarkers by therapeutic area per year.
</p>
</div>
<p>Figure 3 shows the number of biomarkers across all therapeutic areas from 2000 to 2020.</p>
<div class="figure">
<img src="images/Figure_therapeutic_area_number_of_genes.png" alt="Figure 3: Number of biomarkers assessed in Clinical Trials by therapeutic area per year." width="100%" />
<p class="caption">
Figure 3: Number of biomarkers assessed in Clinical Trials by therapeutic area per year.
</p>
</div>
<p>Regarding to the way in which biomarker are assessed, we found that 76% of biomarkers are measured at protein level, 45% at expression level, 30% are genetic biomarkers, 11% are cell markers, 5% are phosphobiomarkers, and 4% are epigenetic (methylation) (Figure 4).</p>
<div class="figure">
<img src="index_files/figure-html/unnamed-chunk-4-1.png" alt="Figure 4: Biomarker types." width="576" />
<p class="caption">
Figure 4: Biomarker types.
</p>
</div>
<p>Most biomarkers (84%) are used to monitor more than one condition. The most common biomarkers measured at protein level are hemoglobin, markers of inflammation, such as CRP, IL6, TNF, IL10, and IFN, enzymes related to liver function such as GPT and AST, and other proteins like insulin, and albumin (Figure 5). AKT1, MAPK1, MTOR, and EGFR are the proteins that are more frequently used as phosphobiomarkers, but EGFR is also frequently assessed for its mutational status, together with KRAS,PIK3CA, PTEN, BRAF, ERBB2, BRCA1 in the context of cancer, and APOE for Alzheimer’s disease. Over 40 clinical trials assess the methylation status of MGMT. The CD4 membrane glycoprotein of T lymphocytes, used to monitor the status of the immune system, is the most frequent cell marker. A selection of the top 100 biomarker-condition pairs can be explored <a href="Pairs.html">here</a></p>
<div class="figure">
<img src="images/Figure_top100bms.png" alt="Figure 5: Top 100 biomarkers and number of conditions." width="80%" />
<p class="caption">
Figure 5: Top 100 biomarkers and number of conditions.
</p>
</div>
<p>Figure 6 shows a selection of biomarkers (50) for Infections (C01), Neoplasms (C04), Cardiovascular diseases (C14), and Mental disorders (F03). Some biomarkers like HBB, and transaminases are universally assessed to monitor the general state of the patient. Nevertheless, other biomarkers are specific to a particular therapeutic area, for example, ACE2 for infections, MGMT, KRAS, BRAF in cancer, VASP in cardiovascular diseases, and OPRM1, APOE, and COMT for mental disorders.</p>
<div class="figure">
<img src="images/Figure_timeline.png" alt="Figure 6: Timeline of biomarkers for a selection of therapeutic areas (C01, Infections, C04, Neoplasms, C14, Cardiovascular diseases, and F03, Mental disorders). The x-axis shows the year of the first trial in which the biomarker is used. Each circle denotes a biomarker, and the name (official Gene Symbol) is provided in different colors. The colors indicate the biomarker type. The size of the circle is proportional to the number of Clinical Trials in which the biomarker is identified. ." width="100%" />
<p class="caption">
Figure 6: Timeline of biomarkers for a selection of therapeutic areas (C01, Infections, C04, Neoplasms, C14, Cardiovascular diseases, and F03, Mental disorders). The x-axis shows the year of the first trial in which the biomarker is used. Each circle denotes a biomarker, and the name (official Gene Symbol) is provided in different colors. The colors indicate the biomarker type. The size of the circle is proportional to the number of Clinical Trials in which the biomarker is identified. .
</p>
</div>
<div id="liver-failure-example" class="section level2">
<h2>Liver Failure example</h2>
<p>In Figure 7 we show an example of the 50 most commonly used biomarkers for liver failure. All biomarkers for liver failure can be explored <a href="Liver.html">here</a></p>
<div class="figure">
<img src="index_files/figure-html/unnamed-chunk-7-1.png" alt="Figure 7: Biomarkers of liver failure." width="768" />
<p class="caption">
Figure 7: Biomarkers of liver failure.
</p>
</div>
</div>
</div>
<div id="conclusions" class="section level1">
<h1>Conclusions</h1>
<p>We have developed and applied a text mining pipeline to extract fine-grained information about biomarkers measured in clinical studies. This pipeline constitutes an efficient way to identify the biomarkers measured in a clinical trial, providing interesting insights both in the type of biomarker and their use across therapeutic areas. Over 3,000 biomarkers associated with 2,600 conditions were identified from 43,000 clinical studies from ClinicalTrials.gov In addition, we have developed a new classification scheme for gene and protein biomarkers, useful to identify the way in which biomarkers are measured in the clinical trial (e.g. genetic, epigenetic, phosphorylation, etc.). Text mining of biomarker information from clinical studies available at CT.gov allows normalization and standardization of the data herein provided, paving the way for automatic and comprehensive data analysis and exploration. Applying this pipeline enables to perform trend analysis on the extracted biomarkers, and their related conditions, assess biomarker specificity, and discover emerging trends about biomarkers of disease. Finally, although there are around 3,000 biomarkers used in the clinical studies, there is still a large fraction of the human genome that remains to be explored as disease biomarkers in the context of clinical studies. Nevertheless, the advances in genomic technologies should facilitate the incorporation of more gene and protein biomarkers into clinical trials.</p>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Bernard2009">
<p>Bernard, Philip S., Joel S. Parker, Michael Mullins, Maggie C.U. Cheung, Samuel Leung, David Voduc, Tammi Vickery, et al. 2009. “Supervised risk predictor of breast cancer based on intrinsic subtypes.” <em>Journal of Clinical Oncology : Official Journal of the American Society of Clinical Oncology</em> 27 (8). J Clin Oncol: 1160–7. <a href="https://doi.org/10.1200/JCO.2008.18.1370" class="uri">https://doi.org/10.1200/JCO.2008.18.1370</a>.</p>
</div>
<div id="ref-Bock2016">
<p>Bock, Christoph, Florian Halbritter, Francisco J. Carmona, Sascha Tierling, Paul Datlinger, Yassen Assenov, María Berdasco, et al. 2016. “Quantitative comparison of DNA methylation assays for biomarker development and clinical applications.” <em>Nature Biotechnology 2016 34:7</em> 34 (7). Nature Publishing Group: 726–37. <a href="https://doi.org/10.1038/nbt.3605" class="uri">https://doi.org/10.1038/nbt.3605</a>.</p>
</div>
<div id="ref-Bodenreider2004">
<p>Bodenreider, Olivier. 2004. “The Unified Medical Language System (UMLS): integrating biomedical terminology.” <em>Nucleic Acids Research</em> 32 (January). Oxford Academic: D267–D270. <a href="https://doi.org/10.1093/NAR/GKH061" class="uri">https://doi.org/10.1093/NAR/GKH061</a>.</p>
</div>
<div id="ref-Gromova2020">
<p>Gromova, Mariya, Annegret Vaggelas, Gabriele Dallmann, and Diane Seimetz. 2020. “Biomarkers: Opportunities and Challenges for Drug Development in the Current Regulatory Landscape.” <em>Biomarker Insights</em> 15 (December). SAGE PublicationsSage UK: London, England: 1177271920974652. <a href="https://doi.org/10.1177/1177271920974652" class="uri">https://doi.org/10.1177/1177271920974652</a>.</p>
</div>
<div id="ref-miura_serum_2008">
<p>Miura, Norimasa, Junichi Hasegawa, and Goshi Shiota. 2008. “Serum Messenger RNA as a Biomarker and Its Clinical Usefulness in Malignancies.” <em>Clinical Medicine. Oncology</em> 2 (September): 511–27. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3161670/" class="uri">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3161670/</a>.</p>
</div>
<div id="ref-Pinero2019">
<p>Piñero, Janet, Juan Manuel Ramírez-Anguita, Josep Saüch-Pitarch, Francesco Ronzano, Emilio Centeno, Ferran Sanz, and Laura I Furlong. 2019. “The DisGeNET knowledge platform for disease genomics: 2019 update.” <em>Nucleic Acids Research</em>, November. <a href="https://doi.org/10.1093/nar/gkz1021" class="uri">https://doi.org/10.1093/nar/gkz1021</a>.</p>
</div>
<div id="ref-Sinnott2021">
<p>Sinnott-Armstrong, Nasa, Yosuke Tanigawa, David Amar, Nina Mars, Christian Benner, Matthew Aguirre, Guhan Ram Venkataraman, et al. 2021. “Genetics of 35 Blood and Urine Biomarkers in the UK Biobank.” <em>Nature Genetics</em> 53 (2): 185–94. <a href="https://doi.org/10.1038/s41588-020-00757-z" class="uri">https://doi.org/10.1038/s41588-020-00757-z</a>.</p>
</div>
<div id="ref-Sorokin2020">
<p>Sorokin, Maxim, Kirill Ignatev, Elena Poddubskaya, Uliana Vladimirova, Nurshat Gaifullin, Dmitriy Lantsov, Andrew Garazha, et al. 2020. “RNA Sequencing in Comparison to Immunohistochemistry for Measuring Cancer Biomarkers in Breast Cancer and Lung Cancer Specimens.” <em>Biomedicines</em> 8 (5): 114. <a href="https://doi.org/10.3390/biomedicines8050114" class="uri">https://doi.org/10.3390/biomedicines8050114</a>.</p>
</div>
<div id="ref-Strimbu2010">
<p>Strimbu, Kyle, and Jorge A. Tavel. 2010. “What are biomarkers?” <em>Current Opinion in HIV and AIDS</em> 5 (6). Curr Opin HIV AIDS: 463–66. <a href="https://doi.org/10.1097/COH.0B013E32833ED177" class="uri">https://doi.org/10.1097/COH.0B013E32833ED177</a>.</p>
</div>
<div id="ref-Tasneem2012">
<p>Tasneem, Asba, Laura Aberle, Hari Ananth, Swati Chakraborty, Karen Chiswell, Brian J. McCourt, and Ricardo Pietrobon. 2012. “The Database for Aggregate Analysis of ClinicalTrials.gov (AACT) and Subsequent Regrouping by Clinical Specialty.” <em>PLOS ONE</em> 7 (3). Public Library of Science: e33677. <a href="https://doi.org/10.1371/JOURNAL.PONE.0033677" class="uri">https://doi.org/10.1371/JOURNAL.PONE.0033677</a>.</p>
</div>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
